packages/webapp/src/components/Noun/Noun.module.css

```diff
@@ -20,13 +21,21 @@
     border-bottom:  1px solid #000;
 }
 
+/* Single Noun */
+.nounWrapper .img {
+    max-width: 500px;
```

Comment:
This is actually just displayed the 320px sized for mobile when I tested it, which is the "natural" size that the Nouns SDK image generator spits out in SVG. I think we'd need to define the width somewhere or the `max-width` won't impact it.